src/commands/integrations/init/integration.ts

```diff
@@ -107,6 +109,7 @@ export default class GenerateIntegrationCommand extends Command {
       scripts: {
         build: "webpack",
         import: "npm run build && prism integrations:import",
+        "generate:manifest": "npm run build && npx @prismatic-io/spectral component-manifest",
```

Comment:
When you get a chance, lets remove this guy. We shouldn't be able to generate component manifest from CNI integrations.